test.js

```diff
@@ -261,6 +261,9 @@ test('rules', function (t) {
   var root = path.join(process.cwd(), 'packages')
   var all = rules(root)
 
+  // TODO: Remove next line
+  all = ["remark-lint-no-duplicate-headings-in-section"]
```

Comment:
on purpose, for faster testing of this change. Let's remove it after discussing if it makes sense :)